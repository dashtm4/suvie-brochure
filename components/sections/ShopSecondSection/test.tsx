import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ShopSecondSection } from '.'

describe('<ShopSecondSection />', () => {
  it('should renders correct <ShopSecondSection />', () => {
    const wrapper = shallow(<ShopSecondSection />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
