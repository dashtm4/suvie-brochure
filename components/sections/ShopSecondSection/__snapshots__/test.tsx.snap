// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShopSecondSection /> should renders correct <ShopSecondSection /> 1`] = `
<section
  className="relative py-20 text-center bg-secondary"
>
  <Wave />
  <div
    className="absolute transform -translate-x-1/2 translate-y-1/2 bottom-full left-1/2"
    role="presentation"
  >
    <Image
      alt=""
      height="150"
      src="/images/100-day-home-trial-risk-free-colored.svg"
      width="150"
    />
  </div>
  <Container>
    <h2
      className="font-serif text-2xl font-normal text-white md:text-4xl"
    >
      <span>
        Try Suvie risk free with a 
      </span>
      <span
        className="inline-flex"
      >
        100-day home trial.
        <InfoLink
          className="ml-1"
          colorHex="#c0c0c0"
          height={23}
          href="/free-trial/"
          width={23}
        />
      </span>
    </h2>
  </Container>
  <Wave
    position="bottom"
  />
</section>
`;
